internal/handler/common/distinct.go

```diff
@@ -41,6 +41,10 @@ type DistinctParams struct {
 
 	Collation *types.Document `ferretdb:"collation,unimplemented"`
 
+	APIVersion           string `ferretdb:"apiVersion,ignored"`
+	APIString            string `ferretdb:"apiStrict,ignored"`
+	APIDeprecationErrors bool   `ferretdb:"apiDeprecationErrors,ignored"`
+
```

Comment:
`distinct` command is not listed in yet https://www.mongodb.com/docs/manual/reference/stable-api-changelog/#std-label-stable-api-changelog. But added it for the consistency